### ISSUE TITLE
Addon weapons do not save to Weapon Loadouts

### ISSUE THREAD

**Issue by CptOrange046** [ProjectFairnessLabs/PF-vMenu#46]
**Describe the bug**
When saving a weapon loadout that has an addon weapon. Then selecting said loadout you do not receive the addon weapon.

**To Reproduce**
1. Get an addon weapon
2. Save your loadout
3. Switch to a loadout without any addon weapons
4. Select the addon weapon loadout
5. See that the addon weapon did not spawn.

**Expected behavior**
You are supposed to receive all weapons within the loadout. Including the addon weapons.

**Additional context**
Any default weapons spawn but if there were any addon weapons they do not spawn.

**Comment by tris-ten** [comment]
How do you have your addon weapons setup with the menu?

**Comment by CptOrange046** [comment]
> 

I have them inside the Addon config. Is there something else I need to do so they save?

**Comment by tris-ten** [comment]
In the json file? Can you screenshot where in the json file you have them in?

**Comment by CptOrange046** [comment]
![image](https://github.com/ProjectFairnessLabs/PF-vMenu/assets/127160864/e92eb5df-a790-49f7-b04e-046113499f9f)

**Comment by tris-ten** [comment]
Yeah, this is a bug. We'll look into a fix for this. @ribbitpoison
